src/librustc_trans/trans/mir/block.rs

```diff
@@ -39,16 +39,17 @@ impl<'bcx, 'tcx> MirContext<'bcx, 'tcx> {
 
         debug!("trans_block: terminator: {:?}", data.terminator());
 
+        // FIXME debuginfo: apply DebugLoc before all uses of `build()`.
```

Comment:
We currently do not emit debug info anywhere in the MIR and there’s a explicit issue for that, so this FIXME is not necessary. Feel free to also remove all the `DebugLoc::None`s you find around as part of this refactoring.